islandora_pathauto.install

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * @file
+ * Implementations of installation hooks.
+ */
+
+/**
+ * Implements hook_uninstall().
+ *
+ * Delete module variables.
+ */
+function islandora_pathauto_uninstall() {
+  $variables = array(
+    'islandora_pathauto_use_cron',
+  );
+  array_walk($variables, 'variable_del');
```

Comment:
Since you are already here, could you please add also `islandora_pathauto_selected_cmodels` variable to the uninstall piece? Not sure if any others missing, but seems like the only one to me

Reply:
@DiegoPino I've added the var to the uninstall hook. Can you please review and let me know if there's anything else. We have a client waiting on this merge. Cheers.